migration/20170328-2-copy-admin-authentication-into-database.py

```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/env python
```

Comment:
Deleted this file because w'ere no longer using AdminAuthenticationService or its table. If the migration sticks around, it'll be a deadend point for the `migrate_database` command.